### ISSUE TITLE
vehicle repair problem

### ISSUE THREAD

**Issue by MosIbOad** [altmp/altv-issues#2386]
### Description of the problem

If vehicle.repair is used at least 1 time for the vehicle to be repaired, even if the tires are flat and the doors are broken, when we change the dimension and go to the dimension of the vehicle again, the tires and doors are fixed and the vehicle is almost repaired again.
If vehicle.repair is never used, there is no problem.

### Reproduction steps

I left a video below that explains the problem completely. At first I created the vehicle and punctured the tires. When I changed the dimension and came back, there was no problem because I did not use vehicle.repair. When I use vehicle.repair, even if the tires of the vehicle are punctured, when we move away from the vehicle and come back, the tires are fixed. This almost happens with other vehicle parts like doors and headlights.

https://streamable.com/wh33qs

### Expected behaviour

.

### Operating system

ubuntu 22.04

### Version

16.2.26 (release)

### Confirmation of issue's presence

- [X] *By submitting this ticket, I affirm that I have verified the presence of this issue on the latest RC (Release Candidate) version available at the time of writing this ticket.*

**Comment by BluePeer** [comment]
thats "broken" since 10+ versions or so
#962 #1013 #1027 #1426 ...

**Comment by xxshady** [comment]
yeah, consider re-creating vehicle but be aware of https://github.com/altmp/altv-js-module/issues/289 (it's not related to JS module)

```js
const newVehicle = new alt.Vehicle(...)

const appearance = oldVehicle.getAppearanceDataBase64()
newVehicle.setAppearanceDataBase64(appearance)

// defer deletion to avoid issue 289
oldVehicle.streamed = false // hide it from players
alt.setTimeout(() => {
    oldVehicle.destroy()
}, 10_000)
```
